tests/conftest.py

```diff
@@ -47,6 +47,11 @@
     pytest.param("gaussian", marks=pytest.mark.gaussian),
 ]
 
+@pytest.fixture(scope="session")
+def test_filename():
+    """Using a filename for the tests."""
+    return "test_config.toml"
```

Comment:
This seems more like test data, so it should be sufficient to have this as a test level variable, rather than as a fixture

Reply:
This was my suggestion (since the same object was being defined independently in several tests).

Upon further consideration, maybe better to just put the data directly in the asserts? e.g., `assert call_history[0][1] == tmpdir.join("test_config.toml")`

Reply:
I've reverted this such that the test data are contained in the tests!